src/main/java/com/gooddata/md/AbstractObj.java

```diff
@@ -27,6 +27,11 @@ protected AbstractObj(@JsonProperty("meta") Meta meta) {
     }
 
     @JsonIgnore
+    public String getId() {
```

Comment:
Please add some javadoc here and to `getIdentifier()` clarifying the semantics of those two IDs, to prevent confusion.

Reply:
done